docs/whats-new/staging-release-notes.md

```diff
@@ -1,5 +1,20 @@
 # Staging release notes
 
+## October 2023
+
+**11 October 2023**
+
+Frontend version: 1.0.0-20231009.1523 | Backend version: 1.69.0-20231006.1929
+
+| Type      | Change      | Description |
+| --- | --- | --- |
+| **Feature** | Check existing users endpoint on Concierge | With the launch of CAM on 23 October 2023, Concierge will now monitor terminated user events. This triggers the cleanup of users in the TechPass directory and sends webhooks to tenants. However, tenants not subscribed to our webhooks might encounter errors. Without these webhook events, tenants will not be able to tidy up datastores effectively, potentially leading to data inconsistencies. |
```

Comment:
The description seems to describe how CAM works and not enough context on what the new API is for.
1. Change should be **Check Existing Users API on Concierge**
2. The API will be used in a one-off event before integrating with TechPass for CAM compliance. The goal is to clean up the tenant's local user store of any accounts that do not exist in TechPass AAD anymore. After integrating with TechPass, the tenants will only receive new user terminated webhook events, that's why this is a vital step.

The description explaining about CAM can stay but add on how the new API brings value.